perfkitbenchmarker/vm_util.py

```diff
@@ -95,6 +98,11 @@ class IpAddressSubset(object):
                   'the receiving VM is reachable by internal IP (REACHABLE), '
                   'external IP only (EXTERNAL) or internal IP only (INTERNAL)')
 
+flags.DEFINE_enum('background_network_iptype', IpAddressSubset.EXTERNAL,
```

Comment:
I'm happy to change it to `background_network_ip_type`.  What's used for specifying ip address for benchmarks like ping and iperf is simply `ip_address`, but I didn't think `background_network_ip_address` would be as clear as `background_network_iptype`